lib/features/public_asset/presentation/public_asset_controller.dart

```diff
@@ -90,6 +93,10 @@ class PublicAssetController extends BaseController {
   void onInit() {
     super.onInit();
     _publicAssetStreamSubscription = _publicAssetStreamGroup.stream.listen(_handleUploadState);
+    BackButtonInterceptor.add((_, __) {
+      discardChanges();
+      return true;
+    }, name: _backButtonInterceptorName);
```

Comment:
this also work for web and mobile too?

Reply:
My bad. The callback should be returning `false`.